install/0000_00_cluster-version-operator_02_service_account.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cvo-dedicated-sa
```

Comment:
nit: can we use `name: cluster-version-operator`?  That avoids the `cvo` acronym, avoids the `sa` echo of the `kind`, and relies on `kubernetes.io/description` to express the dedicated-ness.  The cluster-version-operator-ness is the only aspect we need to keep in the `name` to help avoid colliding with other ServiceAccounts in this namespaces, and the other attributes can be covered in those non-name locations.